source/MaterialXTest/GenShaderUtil.cpp

```diff
@@ -445,8 +530,29 @@ void ShaderGeneratorTester::registerLights(mx::DocumentPtr doc, const std::vecto
     context.getOptions().hwMaxActiveLightSources = lightSourceCount;
 }
 
-void ShaderGeneratorTester::testGeneration(const mx::GenOptions& generateOptions)
+void ShaderGeneratorTester::validate(const mx::GenOptions& generateOptions, const std::string& optionsFilePath)
 {
+    // Test has been turned off so just do nothing.
```

Comment:
Should this code comment still be here?

Reply:
Sorry. Bad grammar. Should read, "if "test has been turned off do nothing.